packages/pilot/src/containers/Withdraw/index.js

```diff
@@ -40,6 +45,27 @@ const setCurrentStep = currentStep => ifElse(
   identity
 )
 
+const buildEmptyState = (onCancel, t) => () => (
```

Comment:
se implementando as modificacões sugeridas nesse comentario https://github.com/pagarme/pilot/pull/1254/files#r293911500, aqui não precisaria retornar uma função
```suggestion
const buildEmptyState = (onCancel, t) => () => (
```

Reply:
resolvido!